pkg/test/framework/test.go

```diff
@@ -69,6 +72,18 @@ func (t *Test) Label(labels ...label.Instance) *Test {
 	return t
 }
 
+// Label applies the given labels to this test.
+func (t *Test) Features(features ...features.Feature) *Test {
+	t.featureLabels = append(t.featureLabels, features...)
+	return t
+}
+
+func (t *Test) NotImplementedYet(features ...features.Feature) *Test {
+	t.Features(features...).
+		Run(func(_ TestContext) { t.goTest.Skip() })
```

Comment:
normally skipped tests have an issue associated. Should this take in an issue?

Reply:
I brought this up with Louis and Sven, and their guidance was that an Issue is premature at this point.  Is there already automation around creating issues for skipped tests?

Reply:
No not automation, just low tech t.Skip("GitHub link..."). I'm just suggesting we pass a string here nothing more. Not a big deal though since we can just add a comment on the test. But it makes sense to have an issue associated with any TODO type things

Reply:
The workflow here is that these stubs should be created at design time, with the stubs (hopefully a decent number of them) representing the test plan.  If they automatically result in issues, I think that's great, but requiring the test plan author to create an issue per test seems onerous, and there is not yet agreement on if or when issues should be created in this process.  If you like, I can add you to the invite the next time we discuss test processes...

Reply:
I never said you need to create an issue per test. You shouldn't just stick TODOs in the code without an issue associated. For example, if I have a bunch of stub tests for WorkloadEntry, I should at least link to an issue "Implement workload entry" to track. I don't think creating an issue to track features should be controversial